core/src/epicli/data/common/defaults/configuration/nfs.yml

```diff
@@ -0,0 +1,12 @@
+kind: configuration/nfs
+title: "NFS Node Config"
+name: default
+specification:
+  nfs_defs:
+    - nfs_path: "/srv/nfs4/exports/"
+      nfs_src: "/exports"
+      nfs_export: "*(rw,sync,no_subtree_check,no_root_squash)"
+  # nfs_defs is a list of NFS definitions to be created
```

Comment:
Can you put the comments behind the actual lines pls? That way when using epicli init they will be included for sure.

Reply:
I have moved the comments now